### ISSUE TITLE
Crash when opening script while Inspector is editing a RichTextLabel's font data

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#59018]
### Godot version

4.0.dev (259114e9e0d5bb01d023ad978e06ed14ca785b1d) (actually a build of #59013)

### System information

Linux, Mageia 9 x86_64

### Issue description

I've got two crashes when trying to open a script (once by creating a new script with the Script Create Dialog and pressing "Create", once by clicking the script icon in the SceneTreeDock for an existing script) while editing a RichTextLabel's font data in the Inspector. Pretty long and messy stacktrace, there's a lot going on.

First crash (pressing "Create" in "Script Create Dialog" to add a new script to a RichTextLabel):
```
================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.alpha.custom_build (fcfd6c5796221029be33095a45aa127d28140cf1)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib64/libc.so.6(+0x403a0) [0x7fd6f772d3a0] (??:0)
[2] /home/akien/Projects/godot/godot.git/bin/godot.linuxbsd.tools.64(__dynamic_cast+0x1d) [0x9a2f80d] (/home/iurt/rpmbuild/BUILD/gcc-12-20220306/obj-x86_64-mageia-linux/x86_64-mageia-linux/libstdc++-v3/libsupc++/../../../../libstdc++-v3/libsupc++/tinfo.h:68)
[3] Button* Object::cast_to<Button>(Object*) (/home/akien/Projects/godot/godot.git/./core/object/object.h:655 (discriminator 1))
[4] EditorPropertyDictionary::_notification(int) (/home/akien/Projects/godot/godot.git/editor/editor_properties_array_dict.cpp:1168 (discriminator 12))
[5] EditorPropertyDictionary::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_properties_array_dict.h:138 (discriminator 14))
[6] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[7] Control::_propagate_theme_changed(Node*, Control*, Window*, bool) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:2575)
[8] Control::_propagate_theme_changed(Node*, Control*, Window*, bool) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:2560)
[9] Control::remove_child_notify(Node*) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:672)
[10] Container::remove_child_notify(Node*) (/home/akien/Projects/godot/godot.git/scene/gui/container.cpp:73)
[11] Node::remove_child(Node*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:1180)
[12] Node::_notification(int) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:157)
[13] Node::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/node.h:45 (discriminator 14))
[14] CanvasItem::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/canvas_item.h:45)
[15] Control::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/control.h:46)
[16] Container::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/container.h:37)
[17] BoxContainer::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/box_container.h:37)
[18] VBoxContainer::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/box_container.h:81)
[19] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[20] Object::_predelete() (/home/akien/Projects/godot/godot.git/core/object/object.cpp:355)
[21] predelete_handler(Object*) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1872)
[22] void memdelete<Node>(Node*) (/home/akien/Projects/godot/godot.git/./core/os/memory.h:104)
[23] EditorInspector::_clear() (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:2981)
[24] EditorInspector::edit(Object*) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3002)
[25] EditorInspector::_notification(int) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3462)
[26] EditorInspector::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.h:423 (discriminator 14))
[27] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[28] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:283)
[29] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[30] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[31] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[32] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[33] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[34] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[35] Node::_set_tree(SceneTree*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:2442)
[36] Node::remove_child(Node*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:1179)
[37] Node::_notification(int) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:157)
[38] Node::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/node.h:45 (discriminator 14))
[39] CanvasItem::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/canvas_item.h:45)
[40] Control::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/control.h:46)
[41] Container::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/container.h:37)
[42] EditorInspectorSection::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.h:260)
[43] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[44] Object::_predelete() (/home/akien/Projects/godot/godot.git/core/object/object.cpp:355)
[45] predelete_handler(Object*) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1872)
[46] void memdelete<Node>(Node*) (/home/akien/Projects/godot/godot.git/./core/os/memory.h:104)
[47] EditorInspector::_clear() (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:2981)
[48] EditorInspector::edit(Object*) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3002)
[49] EditorNode::_edit_current(bool) (/home/akien/Projects/godot/godot.git/editor/editor_node.cpp:2190)
[50] EditorNode::push_item(Object*, String const&, bool) (/home/akien/Projects/godot/godot.git/editor/editor_node.cpp:2112)
[51] SceneTreeDock::_push_item(Object*) (/home/akien/Projects/godot/godot.git/editor/scene_tree_dock.cpp:1342 (discriminator 2))
[52] SceneTreeDock::_script_created(Ref<Script>) (/home/akien/Projects/godot/godot.git/editor/scene_tree_dock.cpp:1929)
[53] void call_with_variant_args_helper<SceneTreeDock, Ref<Script>, 0ul>(SceneTreeDock*, void (SceneTreeDock::*)(Ref<Script>), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:232 (discriminator 6))
[54] void call_with_variant_args<SceneTreeDock, Ref<Script> >(SceneTreeDock*, void (SceneTreeDock::*)(Ref<Script>), Variant const**, int, Callable::CallError&) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:347)
[55] CallableCustomMethodPointer<SceneTreeDock, Ref<Script> >::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/./core/object/callable_method_pointer.h:105)
[56] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/core/variant/callable.cpp:64)
[57] Object::emit_signalp(StringName const&, Variant const**, int) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1087)
[58] Error Object::emit_signal<Ref<Script> >(StringName const&, Ref<Script>) (/home/akien/Projects/godot/godot.git/./core/object/object.h:788)
[59] ScriptCreateDialog::_create_new() (/home/akien/Projects/godot/godot.git/editor/script_create_dialog.cpp:403 (discriminator 5))
[60] ScriptCreateDialog::ok_pressed() (/home/akien/Projects/godot/godot.git/editor/script_create_dialog.cpp:364)
[61] AcceptDialog::_ok_pressed() (/home/akien/Projects/godot/godot.git/scene/gui/dialogs.cpp:104)
[62] void call_with_variant_args_helper<AcceptDialog>(AcceptDialog*, void (AcceptDialog::*)(), Variant const**, Callable::CallError&, IndexSequence<>) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:237 (discriminator 4))
[63] void call_with_variant_args<AcceptDialog>(AcceptDialog*, void (AcceptDialog::*)(), Variant const**, int, Callable::CallError&) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:347)
[64] CallableCustomMethodPointer<AcceptDialog>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/./core/object/callable_method_pointer.h:105)
[65] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/core/variant/callable.cpp:64)
[66] Object::emit_signalp(StringName const&, Variant const**, int) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1087)
[67] Error Object::emit_signal<>(StringName const&) (/home/akien/Projects/godot/godot.git/./core/object/object.h:788)
[68] BaseButton::_pressed() (/home/akien/Projects/godot/godot.git/scene/gui/base_button.cpp:135)
[69] BaseButton::on_action_event(Ref<InputEvent>) (/home/akien/Projects/godot/godot.git/scene/gui/base_button.cpp:176)
[70] BaseButton::gui_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/gui/base_button.cpp:67 (discriminator 3))
[71] Control::_call_gui_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:942)
[72] Viewport::_gui_call_input(Control*, Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/main/viewport.cpp:1303)
[73] Viewport::_gui_input_event(Ref<InputEvent>) (/home/akien/Projects/godot/godot.git/scene/main/viewport.cpp:1603 (discriminator 3))
[74] Viewport::push_input(Ref<InputEvent> const&, bool) (/home/akien/Projects/godot/godot.git/scene/main/viewport.cpp:2697 (discriminator 3))
[75] Window::_window_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/main/window.cpp:974 (discriminator 2))
[76] void call_with_variant_args_helper<Window, Ref<InputEvent> const&, 0ul>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:232 (discriminator 6))
[77] void call_with_variant_args<Window, Ref<InputEvent> const&>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, int, Callable::CallError&) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:347)
[78] CallableCustomMethodPointer<Window, Ref<InputEvent> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/./core/object/callable_method_pointer.h:105)
[79] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/core/variant/callable.cpp:64)
[80] DisplayServerX11::_dispatch_input_event(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/platform/linuxbsd/display_server_x11.cpp:3079)
[81] DisplayServerX11::_dispatch_input_events(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/platform/linuxbsd/display_server_x11.cpp:3049)
[82] Input::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/home/akien/Projects/godot/godot.git/core/input/input.cpp:660)
[83] Input::flush_buffered_events() (/home/akien/Projects/godot/godot.git/core/input/input.cpp:878)
[84] DisplayServerX11::process_events() (/home/akien/Projects/godot/godot.git/platform/linuxbsd/display_server_x11.cpp:4039)
[85] OS_LinuxBSD::run() (/home/akien/Projects/godot/godot.git/platform/linuxbsd/os_linuxbsd.cpp:355)
[86] /home/akien/Projects/godot/godot.git/bin/godot.linuxbsd.tools.64(main+0x136) [0x55c5f5c] (/home/akien/Projects/godot/godot.git/platform/linuxbsd/godot_linuxbsd.cpp:60)
[87] /lib64/libc.so.6(+0x2d177) [0x7fd6f771a177] (??:0)
[88] /lib64/libc.so.6(__libc_start_main+0x85) [0x7fd6f771a235] (??:0)
[89] /home/akien/Projects/godot/godot.git/bin/godot.linuxbsd.tools.64(_start+0x21) [0x55c5d61] (??:?)
-- END OF BACKTRACE --
================================================================
```

Second crash: attempting to open the script of a parent Control node while editing one of its RichTextLabel children:
```
================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.alpha.custom_build (fcfd6c5796221029be33095a45aa127d28140cf1)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib64/libc.so.6(+0x403a0) [0x7f4bc4fa33a0] (??:0)
[2] /home/akien/Projects/godot/godot.git/bin/godot.linuxbsd.tools.64(__dynamic_cast+0x1d) [0x9a2f80d] (/home/iurt/rpmbuild/BUILD/gcc-12-20220306/obj-x86_64-mageia-linux/x86_64-mageia-linux/libstdc++-v3/libsupc++/../../../../libstdc++-v3/libsupc++/tinfo.h:68)
[3] Button* Object::cast_to<Button>(Object*) (/home/akien/Projects/godot/godot.git/./core/object/object.h:655 (discriminator 1))
[4] EditorPropertyDictionary::_notification(int) (/home/akien/Projects/godot/godot.git/editor/editor_properties_array_dict.cpp:1168 (discriminator 12))
[5] EditorPropertyDictionary::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_properties_array_dict.h:138 (discriminator 14))
[6] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[7] Control::_propagate_theme_changed(Node*, Control*, Window*, bool) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:2575)
[8] Control::_propagate_theme_changed(Node*, Control*, Window*, bool) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:2560)
[9] Control::remove_child_notify(Node*) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:672)
[10] Container::remove_child_notify(Node*) (/home/akien/Projects/godot/godot.git/scene/gui/container.cpp:73)
[11] Node::remove_child(Node*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:1180)
[12] Node::_notification(int) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:157)
[13] Node::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/node.h:45 (discriminator 14))
[14] CanvasItem::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/canvas_item.h:45)
[15] Control::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/control.h:46)
[16] Container::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/container.h:37)
[17] BoxContainer::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/box_container.h:37)
[18] VBoxContainer::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/box_container.h:81)
[19] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[20] Object::_predelete() (/home/akien/Projects/godot/godot.git/core/object/object.cpp:355)
[21] predelete_handler(Object*) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1872)
[22] void memdelete<Node>(Node*) (/home/akien/Projects/godot/godot.git/./core/os/memory.h:104)
[23] EditorInspector::_clear() (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:2981)
[24] EditorInspector::edit(Object*) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3002)
[25] EditorInspector::_notification(int) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3462)
[26] EditorInspector::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.h:423 (discriminator 14))
[27] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[28] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:283)
[29] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[30] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[31] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[32] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[33] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[34] Node::_propagate_exit_tree() (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:272 (discriminator 3))
[35] Node::_set_tree(SceneTree*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:2442)
[36] Node::remove_child(Node*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:1179)
[37] Node::_notification(int) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:157)
[38] Node::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/node.h:45 (discriminator 14))
[39] CanvasItem::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/canvas_item.h:45)
[40] Control::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/control.h:46)
[41] Container::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/container.h:37)
[42] EditorInspectorSection::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.h:260)
[43] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[44] Object::_predelete() (/home/akien/Projects/godot/godot.git/core/object/object.cpp:355)
[45] predelete_handler(Object*) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1872)
[46] void memdelete<Node>(Node*) (/home/akien/Projects/godot/godot.git/./core/os/memory.h:104)
[47] EditorInspector::_clear() (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:2981)
[48] EditorInspector::edit(Object*) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3002)
[49] EditorNode::_edit_current(bool) (/home/akien/Projects/godot/godot.git/editor/editor_node.cpp:2190)
[50] EditorNode::push_item(Object*, String const&, bool) (/home/akien/Projects/godot/godot.git/editor/editor_node.cpp:2112)
[51] InspectorDock::_resource_selected(Ref<Resource> const&, String const&) (/home/akien/Projects/godot/godot.git/editor/inspector_dock.cpp:366)
[52] InspectorDock::edit_resource(Ref<Resource> const&) (/home/akien/Projects/godot/godot.git/editor/inspector_dock.cpp:444 (discriminator 2))
[53] EditorNode::edit_resource(Ref<Resource> const&) (/home/akien/Projects/godot/godot.git/./editor/editor_node.h:740)
[54] SceneTreeDock::_script_open_request(Ref<Script> const&) (/home/akien/Projects/godot/godot.git/editor/scene_tree_dock.cpp:1337 (discriminator 2))
[55] void call_with_variant_args_helper<SceneTreeDock, Ref<Script> const&, 0ul>(SceneTreeDock*, void (SceneTreeDock::*)(Ref<Script> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:232 (discriminator 6))
[56] void call_with_variant_args<SceneTreeDock, Ref<Script> const&>(SceneTreeDock*, void (SceneTreeDock::*)(Ref<Script> const&), Variant const**, int, Callable::CallError&) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:347)
[57] CallableCustomMethodPointer<SceneTreeDock, Ref<Script> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/./core/object/callable_method_pointer.h:105)
[58] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/core/variant/callable.cpp:64)
[59] Object::emit_signalp(StringName const&, Variant const**, int) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1087)
[60] Error Object::emit_signal<Ref<Script> >(StringName const&, Ref<Script>) (/home/akien/Projects/godot/godot.git/./core/object/object.h:788)
[61] SceneTreeEditor::_cell_button_pressed(Object*, int, int) (/home/akien/Projects/godot/godot.git/editor/scene_tree_editor.cpp:76 (discriminator 5))
[62] void call_with_variant_args_helper<SceneTreeEditor, Object*, int, int, 0ul, 1ul, 2ul>(SceneTreeEditor*, void (SceneTreeEditor::*)(Object*, int, int), Variant const**, Callable::CallError&, IndexSequence<0ul, 1ul, 2ul>) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:237 (discriminator 4))
[63] void call_with_variant_args<SceneTreeEditor, Object*, int, int>(SceneTreeEditor*, void (SceneTreeEditor::*)(Object*, int, int), Variant const**, int, Callable::CallError&) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:347)
[64] CallableCustomMethodPointer<SceneTreeEditor, Object*, int, int>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/./core/object/callable_method_pointer.h:105)
[65] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/core/variant/callable.cpp:64)
[66] Object::emit_signalp(StringName const&, Variant const**, int) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1087)
[67] Error Object::emit_signal<TreeItem*, int, int>(StringName const&, TreeItem*, int, int) (/home/akien/Projects/godot/godot.git/./core/object/object.h:788)
[68] Tree::gui_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/gui/tree.cpp:3321)
[69] Control::_call_gui_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:942)
[70] Viewport::_gui_call_input(Control*, Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/main/viewport.cpp:1303)
[71] Viewport::_gui_input_event(Ref<InputEvent>) (/home/akien/Projects/godot/godot.git/scene/main/viewport.cpp:1603 (discriminator 3))
[72] Viewport::push_input(Ref<InputEvent> const&, bool) (/home/akien/Projects/godot/godot.git/scene/main/viewport.cpp:2697 (discriminator 3))
[73] Window::_window_input(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/scene/main/window.cpp:974 (discriminator 2))
[74] void call_with_variant_args_helper<Window, Ref<InputEvent> const&, 0ul>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, Callable::CallError&, IndexSequence<0ul>) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:232 (discriminator 6))
[75] void call_with_variant_args<Window, Ref<InputEvent> const&>(Window*, void (Window::*)(Ref<InputEvent> const&), Variant const**, int, Callable::CallError&) (/home/akien/Projects/godot/godot.git/./core/variant/binder_common.h:347)
[76] CallableCustomMethodPointer<Window, Ref<InputEvent> const&>::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/./core/object/callable_method_pointer.h:105)
[77] Callable::call(Variant const**, int, Variant&, Callable::CallError&) const (/home/akien/Projects/godot/godot.git/core/variant/callable.cpp:64)
[78] DisplayServerX11::_dispatch_input_event(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/platform/linuxbsd/display_server_x11.cpp:3079)
[79] DisplayServerX11::_dispatch_input_events(Ref<InputEvent> const&) (/home/akien/Projects/godot/godot.git/platform/linuxbsd/display_server_x11.cpp:3049)
[80] Input::_parse_input_event_impl(Ref<InputEvent> const&, bool) (/home/akien/Projects/godot/godot.git/core/input/input.cpp:660)
[81] Input::flush_buffered_events() (/home/akien/Projects/godot/godot.git/core/input/input.cpp:878)
[82] DisplayServerX11::process_events() (/home/akien/Projects/godot/godot.git/platform/linuxbsd/display_server_x11.cpp:4039)
[83] OS_LinuxBSD::run() (/home/akien/Projects/godot/godot.git/platform/linuxbsd/os_linuxbsd.cpp:355)
[84] /home/akien/Projects/godot/godot.git/bin/godot.linuxbsd.tools.64(main+0x136) [0x55c5f5c] (/home/akien/Projects/godot/godot.git/platform/linuxbsd/godot_linuxbsd.cpp:60)
[85] /lib64/libc.so.6(+0x2d177) [0x7f4bc4f90177] (??:0)
[86] /lib64/libc.so.6(__libc_start_main+0x85) [0x7f4bc4f90235] (??:0)
[87] /home/akien/Projects/godot/godot.git/bin/godot.linuxbsd.tools.64(_start+0x21) [0x55c5d61] (??:?)
-- END OF BACKTRACE --
================================================================
```

### Steps to reproduce

Not sure yet about precise steps. I'll attach my test project, and update this section if/when I find how to reproduce this reliably.

### Minimal reproduction project

[TestRTLEffectsArabic2.zip](https://github.com/godotengine/godot/files/8230747/TestRTLEffectsArabic2.zip)
(23 MB, several big fonts)

**Comment by akien-mga** [comment]
Another crash, this time while editing Import properties for a TTF from that project. Couldn't reproduce yet.
```
================================================================
handle_crash: Program crashed with signal 11
Engine version: Godot Engine v4.0.alpha.custom_build (fcfd6c5796221029be33095a45aa127d28140cf1)
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib64/libc.so.6(+0x403a0) [0x7f3b4fa263a0] (??:0)
[2] godot-git(__dynamic_cast+0x1d) [0x9a2f80d] (/home/iurt/rpmbuild/BUILD/gcc-12-20220306/obj-x86_64-mageia-linux/x86_64-mageia-linux/libstdc++-v3/libsupc++/../../../../libstdc++-v3/libsupc++/tinfo.h:68)
[3] Button* Object::cast_to<Button>(Object*) (/home/akien/Projects/godot/godot.git/./core/object/object.h:655 (discriminator 1))
[4] EditorPropertyDictionary::_notification(int) (/home/akien/Projects/godot/godot.git/editor/editor_properties_array_dict.cpp:1168 (discriminator 12))
[5] EditorPropertyDictionary::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_properties_array_dict.h:138 (discriminator 14))
[6] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[7] Control::_propagate_theme_changed(Node*, Control*, Window*, bool) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:2575)
[8] Control::_propagate_theme_changed(Node*, Control*, Window*, bool) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:2560)
[9] Control::remove_child_notify(Node*) (/home/akien/Projects/godot/godot.git/scene/gui/control.cpp:672)
[10] Container::remove_child_notify(Node*) (/home/akien/Projects/godot/godot.git/scene/gui/container.cpp:73)
[11] Node::remove_child(Node*) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:1180)
[12] Node::_notification(int) (/home/akien/Projects/godot/godot.git/scene/main/node.cpp:157)
[13] Node::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/node.h:45 (discriminator 14))
[14] CanvasItem::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/main/canvas_item.h:45)
[15] Control::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/control.h:46)
[16] Container::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/container.h:37)
[17] BoxContainer::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/box_container.h:37)
[18] VBoxContainer::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/./scene/gui/box_container.h:81)
[19] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[20] Object::_predelete() (/home/akien/Projects/godot/godot.git/core/object/object.cpp:355)
[21] predelete_handler(Object*) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:1872)
[22] void memdelete<Node>(Node*) (/home/akien/Projects/godot/godot.git/./core/os/memory.h:104)
[23] EditorInspector::_clear() (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:2981)
[24] EditorInspector::update_tree() (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:2413)
[25] EditorInspector::_notification(int) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.cpp:3493)
[26] EditorInspector::_notificationv(int, bool) (/home/akien/Projects/godot/godot.git/editor/editor_inspector.h:423 (discriminator 14))
[27] Object::notification(int, bool) (/home/akien/Projects/godot/godot.git/core/object/object.cpp:830)
[28] SceneTree::_notify_group_pause(StringName const&, int) (/home/akien/Projects/godot/godot.git/scene/main/scene_tree.cpp:851)
[29] SceneTree::process(double) (/home/akien/Projects/godot/godot.git/scene/main/scene_tree.cpp:452)
[30] Main::iteration() (/home/akien/Projects/godot/godot.git/main/main.cpp:2675)
[31] OS_LinuxBSD::run() (/home/akien/Projects/godot/godot.git/platform/linuxbsd/os_linuxbsd.cpp:357)
[32] godot-git(main+0x136) [0x55c5f5c] (/home/akien/Projects/godot/godot.git/platform/linuxbsd/godot_linuxbsd.cpp:60)
[33] /lib64/libc.so.6(+0x2d177) [0x7f3b4fa13177] (??:0)
[34] /lib64/libc.so.6(__libc_start_main+0x85) [0x7f3b4fa13235] (??:0)
[35] godot-git(_start+0x21) [0x55c5d61] (??:?)
-- END OF BACKTRACE --
================================================================
```

**Comment by akien-mga** [comment]
This has likely been fixed by #58751 with this change: https://github.com/godotengine/godot/pull/58751/files#diff-17781aa265945f5f8988f4d66150a04b73fcdd99f5600b24409d731d24e6fbf1R1147

Closing, will reopen if I can reproduce it again.
